tests/functional/spec/features/repeating_blocks/list_collector_repeating_blocks.spec.js

```diff
@@ -71,14 +72,15 @@ describe("List Collector Repeating Blocks", () => {
       await click(AnyOtherTradingDetailsPage.submit());
       await click(SectionCompaniesPage.submit());
       await click(SubmitPage.submit());
+      await expect(await browser.getUrl()).toContain(ThankYouPage.pageName);
```

Comment:
Would it be better for the test to be `await expect(browser).toHaveUrlContaining(ThankYouPage.pageName);` to match the new formatting of the tests?

Reply:
Was awaiting merge of your PR and the inevitable merge conflicts before switching to use those 👍